src/dto/test_results/TestResult.ts

```diff
@@ -1,7 +1,9 @@
 import {DtoObject} from '../DtoObject';
 import {TestResultBuildAttributes} from './TestResultBuildAttributes';
 import {TestResultTestField} from './TestResultTestField';
-import {TestResultTestRunElement} from './TestResultTestRunElement';
+import {TestResultUnitTestRunElement} from './TestResultUnitTestRunElement';
```

Comment:
What are these import statements for if no other code change here that uses this? Or is it GitHub not showing this properly?